relationshiptypes.md

```diff
@@ -68,3 +68,51 @@ A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspac
 ```
 
 The 2 [item types](itemtypes.md) are embedded
+
+## Search methods
+
+### Relationship types containing an entity type
+**/api/core/relationshiptypes/search/byEntityType?type=<:entity-type-label>**
+
+Parameters:
+* The `type` should be the entity type label from the [entity types endpoint](entitytypes.md). It is mandatory. It can occur on either the left or right hand side
+
+A sample search would be /server/api/core/relationshiptypes/search/byEntityType?id=Publication
```

Comment:
Tiny thing to change. This example is using an `id=` param instead of `type=` param.

Also, I think this example should say `type=Person`, as that would ensure it matches the response example below...as in your example response, both relationshiptypes returned are associated with a `Person` entity.

Reply:
i have updated the example to reflect a search for Publication. It is indeed more significant than the one by Person because according to our default `relationship-types.xml` there are two relationshiptype that involve publication having exactly the same labels. One is between person and publication and the other between orgunit and publication, this help us also to remember why the two labels are not sufficient to identify a single relationship type